app/routers/quizzes.py

```diff
@@ -7,8 +7,9 @@
 router = APIRouter(prefix="/quiz", tags=["Quiz"])
 
 
-@router.post("/", response_model=QuizResponse)
+@router.post("/")
 async def create_quiz(quiz: Quiz):
+    """Returns the ID of created quiz in a dictionary with key as 'quiz_id'"""
     quiz = jsonable_encoder(quiz)
     new_quiz = client.quiz.quizzes.insert_one(quiz)
     created_quiz = client.quiz.quizzes.find_one({"_id": new_quiz.inserted_id})
```

Comment:
Why do we need this step anymore?

Reply:
Well you're correct in a way. This step is not needed anymore because we dont' need to return the whole quiz from the function's response but there are some gymnastics being applied to this returned quiz in the steps below. Basically just putting `question_set_id` inside the questions is needed and for that, created quiz is needed